kimchi/src/prover.rs

```diff
@@ -1050,7 +1051,8 @@ where
 
         let polys = prev_challenges
             .iter()
-            .map(|(chals, comm)| {
+            .map(|c| {
+                let RecursionChallenge { chals, comm } = c;
```

Comment:
Nit: inline this pattern in the argument?

Reply:
Thanks!